tests/test_solution.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+from pathlib import Path
+
+import numpy as np
+import pytest
+import xarray
+from helpers import dataclasses_are_equal
+
+from openlifu import Point, Pulse, Sequence, Solution
+
+
+@pytest.fixture()
+def example_solution() -> Solution:
```

Comment:
Why not have this into a file, like you did for [tests/test_database.py](tests/test_database.py) ?

Reply:
The goal of these tests is to test the functionality of the Solution class features independently, not necessarily file i/o. On the other hand `test_database` is focused on the file i/o aspect.